src/ffi.rs

```diff
@@ -219,6 +220,28 @@ pub fn object_to_shared_lib(object: &[u8], output_filename: &Path) -> Result<()>
         }
     }
 
+    let runtime_library_path = if let Ok(extra_dir) = std::env::var("CAIRO_NATIVE_RUNTIME_LIBRARY")
+    {
+        let path = Path::new(&extra_dir);
+        if path.is_absolute() {
+            extra_dir
+        } else {
+            let mut absolute_path = env::current_dir()
+                .expect("Failed to get the current directory")
+                .join(path);
+            absolute_path = absolute_path
+                .canonicalize()
+                .expect("Failed to cannonicalize path");
+            String::from(
+                absolute_path
+                    .to_str()
+                    .expect("Absolute path contains non-utf8 characters"),
+            )
+        }
+    } else {
+        String::from("libcario_native_runtime.a")
```

Comment:
There is a typo here: `libcario_native_runtime` should be `libcairo_native_runtime`

Reply:
Fixed, can you please approve the workflow, please?

Reply:
> Fixed, can you please approve the workflow, please?

done